backend/utils.py

```
@@ -0,0 +1,8 @@
+from models import Setup
+from config import config
+
+def same_key(key: str):
+    key1 = config.get('Global', 'BINDING_KEY')
+    if key1 == key:
+        return True
+    return False
```

Ruff (0.11.9)

Return the condition `key1 == key` directly

Replace with `return key1 == key`

(SIM103)